core-it-support/core-it-bootstrap/pom.xml

```diff
@@ -19,45 +19,58 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-<project>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.apache.maven.its.bootstrap</groupId>
-    <artifactId>maven-it-boostrap</artifactId>
-    <version>1.0</version>
+    <groupId>org.apache.maven.its</groupId>
+    <artifactId>core-it-support</artifactId>
+    <version>2.1-SNAPSHOT</version>
   </parent>
 
-  <groupId>org.apache.maven.its.bootstrap</groupId>
-  <artifactId>group-13</artifactId>
-  <version>1.0</version>
-  <packaging>jar</packaging>
-
-  <name>Maven Integration Test :: Boostrap :: Group-13</name>
-
-  <properties>
-  </properties>
+  <artifactId>core-it-bootstrap</artifactId>
+  <packaging>maven-plugin</packaging>
+  <name>Bootstrap plugin</name>
 
   <dependencies>
-    <dependency>
-      <groupId>org.codehaus.plexus</groupId>
-      <artifactId>plexus-utils</artifactId>
-      <version>2.0.5</version>
-    </dependency>
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-plugin-api</artifactId>
-      <version>3.1.0</version>
+      <version>3.8.6</version>
     </dependency>
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-core</artifactId>
-      <version>3.1.0</version>
+      <version>3.8.6</version>
     </dependency>
     <dependency>
-      <groupId>org.apache.maven.plugins</groupId>
-      <artifactId>maven-plugin-plugin</artifactId>
-      <version>3.6.0</version>
+      <groupId>org.apache.maven.plugin-tools</groupId>
+      <artifactId>maven-plugin-annotations</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven.resolver</groupId>
+      <artifactId>maven-resolver-impl</artifactId>
+      <version>1.8.0</version>
```

Comment:
Attention, Maven 3.8.6 does not use this API version.

Reply:
Yeah, though given this branch is for JDK8 (hence , I would expect we raise the maven version to 3.9.0 as soon as it's out, so using resolver 1.8.x.  But I'll upgrade it to 1.8.1.

Reply:
Agreed. Also note that Resolver 1.8.x has slight API incompat. Likely won't be here an issue.